tools/AutoTuner/helm-chart/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for Ray.
```

Comment:
The docs here : https://docs.ray.io/en/latest/cluster/quickstart.html#launch-a-cluster-on-a-cloud-provider are easy to read and helpful. Suggestions: Include links to Ray docs such as this section.

Reply:
They do not directly correlate. The documentation you pointed out is to deploy a Ray cluster on GCP without Kubernetes.